jplag.endtoend-testing/src/test/java/de/jplag/endtoend/EndToEndSuiteTest.java

```diff
@@ -60,8 +60,8 @@ public EndToEndSuiteTest() throws IOException {
      */
     private void setRunOptions() {
         options = new ArrayList<>();
-        options.add(new Options(1));
-        options.add(new Options(15));
+        options.add(new Options(9));
+        options.add(new Options(3));
     }
```

Comment:
I still don't understand why this has to be specified in code in addition to the JSON?

Reply:
The JSON file is only stored for the results of the tests and not for the creation of test cases. It is not intended to load information from the JSON data to generate test cases. The test cases are generated in the code and the results are stored in the JSON files. This is necessary to be able to recognize the results of the last change in the code.